staging/src/k8s.io/kubectl/pkg/cmd/top/top_node.go

```diff
@@ -91,6 +92,7 @@ func NewCmdTopNode(f cmdutil.Factory, o *TopNodeOptions, streams genericclioptio
 	cmd.Flags().StringVar(&o.SortBy, "sort-by", o.SortBy, "If non-empty, sort nodes list using specified field. The field can be either 'cpu' or 'memory'.")
 	cmd.Flags().BoolVar(&o.NoHeaders, "no-headers", o.NoHeaders, "If present, print output without headers")
 	cmd.Flags().BoolVar(&o.UseProtocolBuffers, "use-protocol-buffers", o.UseProtocolBuffers, "Enables using protocol-buffers to access Metrics API.")
+	cmd.Flags().BoolVar(&o.Capacity, "capacity", o.Capacity, "Print node resources based on Capacity instead of Allocatable(default) of the nodes.")
```

Comment:
maybe `--show-capacity` instead?

Reply:
@soltysh Thank you for your review. I've modified it along with the related variable names.

Reply:
The variable also had replaced "Capacity" with "ShowCapacity".